packages/server/src/module/transpile.rs

```diff
@@ -15,14 +15,14 @@ pub struct Error {
 }
 
 impl Server {
-	pub fn transpile_module(text: String) -> tg::Result<Output> {
+	pub fn transpile_module(text: String, file: tg::error::File) -> tg::Result<Output> {
```

Comment:
passing a tg::error::File here is obviously super weird. @nitsky thoughts on if we should create a new type here? When printing the error we need to know the source file not just the text. Because this was missing, I believe these errors were actually incorrectly being marked as "internal" which is silly.